xarray_sentinel/sentinel1.py

```diff
@@ -172,3 +200,10 @@ def guess_can_open(self, filename_or_obj: T.Any) -> bool:
         except TypeError:
             return False
         return ext.lower() in {".safe"}
+
+
+OPENERS = {
```

Comment:
Naming. These are not *all* the `OPENERS`. Better something like
```suggestion
METADATA_OPENERS = {
```